### ISSUE TITLE
Cannot even install the bit?

### ISSUE THREAD

**Issue by iamrommel** [teambit/bit#3985]
### Describe the bug

Cannot install via npm install bit-bin -g

It was stuck on this part
![image](https://user-images.githubusercontent.com/177265/112362954-f63ab300-8d0f-11eb-9634-8985a964015a.png)

### Specifications

- Bit version: ??
- Node version: 14
- npm / yarn version: npm7
- Platform: Apple M1 
- Bit compiler (include version): ??
- Bit tester (include version):  ??

**Comment by davidfirst** [comment]
@iamrommel , I'm curious what was it :)

**Comment by GiladShoham** [comment]
My guess is something related to the m1 chip which cause something to be pretty slow during node-gyp

**Comment by iamrommel** [comment]
I'm not sure if NPM works (e.g. npm install bit-bin --global) but when i do brew install bit it works fine.

The documation on **quick start** say i do npm install, but when i travel to the next page, it say i can do home brew, I feel that is misleading or something?
![image](https://user-images.githubusercontent.com/177265/112376830-e88d2980-8d1f-11eb-9590-0ca362b4f6f9.png)
![image](https://user-images.githubusercontent.com/177265/112376958-1d00e580-8d20-11eb-833c-0f40a8d6b23c.png)

Here is also the NPM logs, when i first reported
![debug](https://user-images.githubusercontent.com/177265/112377338-94367980-8d20-11eb-9067-782e9c7f0077.png)


I just did do homebrew and I was able to do bit things now, also decided to use the harmony in exploring this project.
